advancedcontentfilter/doc/advancedcontentfilter.md

```diff
@@ -34,6 +34,14 @@ If the expression using a post variables returns `true`, the post will be collap
 
 A post will be collapsed if at least one rule matches, but all matching rule names will be displayed above the collapsed content.
 
+## Examples
+
+1. To block specific domains `body matches "/\\.example\\.com/"`
+2. To block everything that contains the words `body matches "/Guten Morgen/"`
+3. To block every occurence of the word facebook with a space in front and after the word `body matches "//s facebook/s /"`
```

Comment:
Are you sure it's `/s`? I feel like you're expecting the "any whitespace" character class but it would be `\s`, and in this case, possibly `\\s` because of the way we're entering the regular expression.